src/AutoRest.CSharp.V3/Generation/Writers/ClientWriter.cs

```diff
@@ -340,16 +340,19 @@ private void WritePagingOperation(CodeWriter writer, Paging pagingMethod, bool a
             }
         }
 
-        private CodeWriterDelegate WriteConstantOrParameter(ParameterOrConstant constantOrParameter) => writer =>
+        private CodeWriterDelegate WriteConstantOrParameter(ParameterOrConstant constantOrParameter, bool ignoreNullability = false) => writer =>
         {
             if (constantOrParameter.IsConstant)
             {
                 WriteConstant(writer, constantOrParameter.Constant);
             }
             else
             {
-                writer.AppendRaw(constantOrParameter.Parameter.Name)
-                    .AppendNullableValue(_typeFactory.CreateType(constantOrParameter.Type));
+                writer.AppendRaw(constantOrParameter.Parameter.Name);
+                if (!ignoreNullability)
+                {
+                    writer.AppendNullableValue(_typeFactory.CreateType(constantOrParameter.Type));
```

Comment:
This was for the thing where we had nullable parameters for the CreateRequest method. In that situation, we decided to just ignore the nullability aspect of it.